.vsts-ci.yml

```diff
@@ -206,6 +206,14 @@ jobs:
             # Do not publish zips and tarballs. The linux-x64 binaries are
             # already published by Build_LinuxPortable_Release_x64
             _AdditionalBuildParameters: '/p:PublishBinariesAndBadge=false'
+          Build_Internal_Package_X64:
```

Comment:
We have a leg that updates the latest packages in blob storage only once and only when all the other legs have passed. Shouoldn't the same thing be done here? We would not want this packge to be published if any of the other legs have failed.

Reply:
Could you elaborate on that so I can get my head around it? All the other legs are publishing their packages to the same dotnetcli feed (hence the build break, this package was the only one that didn't get generated with a different name for each leg, so it failed during publishing on every leg except the first that attempted it)

This package will still not be available in BAR, or in the final feed location unless all the other legs succeeded.

Did I understand that wrong?

Reply:
Ah. Then I misunderstood. I thought this would go to BAR at this step. If that's not the case, then we should be ok.

However, I would suggest you look at https://github.com/dotnet/core-sdk/tree/master/src/CopyToLatest to see how do we do this without treating it as a leg in the middle of the build.

Reply:
Hmm, From what I understand, this package still needs to be built and pushed to the intermediate feed so that CopyToLatest picks it up and pushes it to the final location. 

I thought about making it its own leg because I thought it would make sense to have it built independently as it's not a product package, and it avoids the multiple publishes that are causing the problem.

In your eyes what would be the correct place to build/publish this package if it's not as a separate leg?

I can think of the following options, but none of them seem ideal:
* Create MSBuild targets that build and push the package to the feed, and hook them up so they run before CopyToLatest does. I'd rather keep having the package being managed by Arcade's build scripts, since they know how to do the right thing. 
* Add custom steps that will look very similar to what eng/build.yml already does, and stick those steps to the CopyToLatest leg of the build. This way I can keep using Arcade to build and publish to the intermediate feed, there won't be any extra legs just for one package, but it will probably cause some repetition in the code.

Reply:
While we sort this out. I created #1142 to just remove the project from the main SLN file. That will fix the build break while I come up with a better solution.